### ISSUE TITLE
Rotation of objects doesn't work correctly

### ISSUE THREAD

**Issue by eAi** [wonderunit/storyboarder#1845]
**Describe the bug**
Rotation of objects doesn't behave as expected. This may be a case of gimbal lock? It seems that Y rotation is in local space but X and Z are in world space - but this may be a misreading of the situation. I've looked through the code a bit, but I can't find where this aspect is handled.

**To Reproduce**
1. Create an object (not a cube, something with a distinctive rotation)
2. Set the Z rotation in the inspector to 90°
3. Set the "Rotate X" value to 90°
4. Observe the orientation of the object
5. Set the "Rotate X" back to 0°
6. Set the "Rotate Y" value to 90°
7. Observe that the orientation of the object is the same as seen in Step 4.

**Expected behavior**
I would expect the rotation to be in degrees around each axis of the object in local space. With this bug it is impossible, or very difficult to achieve certain rotations. 

**Desktop (please complete the following information):**
 - OS: macOS 10.14.6
 - Storyboarder version: 1.15.0

**Comment by setpixel** [comment]
Yes - this is annoying. The Rotate X should really be labeled X Rotation, because it is just the X rotation component. What should be there is a gizmo that allows global rotation of objects so you can easily reason about how you want to rotate them. In VR, this is easy. You just pick it up and rotate it as you want. 

We are rewriting the viewport of Shot Generator to be much faster and create a lot of the interface gizmos that are needed to interact with the 2d viewport. Sorry about that!
